core-web/libs/data-access/src/lib/dot-edit-page-resolver/dot-edit-page-resolver.service.ts

```diff
@@ -50,27 +50,20 @@ export class DotEditPageResolver implements Resolve<DotPageRenderState> {
     ): Observable<DotPageRenderState> {
         if (!dotRenderedPageState.page.canEdit) {
             return throwError(
-                new HttpErrorResponse(
-                    new HttpResponse<DotCMSResponse<DotPageRenderState>>({
-                        body: null,
-                        status: HttpCode.FORBIDDEN,
-                        headers: null,
-                        url: ''
-                    })
-                )
+                new HttpErrorResponse({
+                    status: HttpCode.FORBIDDEN,
+                    url: ''
+                })
             );
         } else if (!dotRenderedPageState.layout) {
             return throwError(
-                new HttpErrorResponse(
-                    new HttpResponse<DotCMSResponse<DotPageRenderState>>({
-                        body: null,
-                        status: HttpCode.FORBIDDEN,
-                        headers: new HttpHeaders({
-                            'error-key': 'dotcms.api.error.license.required'
-                        }),
-                        url: ''
-                    })
-                )
+                new HttpErrorResponse({
+                    status: HttpCode.FORBIDDEN,
+                    headers: new HttpHeaders({
+                        'error-key': 'dotcms.api.error.license.required'
+                    }),
```

Comment:
If the page have an advaced template it will not have `.layout` either... so we need to check that as well, maybe we can look at the `template` property and see if there is a property that tell us that, maybe the `drawed` property.

Reply:
I didn't change this at all. The `eslint` and `typescript` were throwing an error because we were doing this:

```typescript
new HttpErrorResponse(
    new HttpResponse<DotCMSResponse<DotPageRenderState>>({
        body: null,
        status: HttpCode.FORBIDDEN,
        headers: new HttpHeaders({
            error-key': 'dotcms.api.error.license.required'
        }),
        url: ''
    })
);
```

We can not pass a `new HttpResponse` in the `new HttpErrorResponse` response constructor